src/xcm/mod.rs

```diff
@@ -41,10 +41,12 @@ impl<T: Config> Pallet<T> {
 	) -> Result<(), Error<T>> {
 		let interior = X1(PalletInstance(Pallet::<T>::index() as u8));
 		let dest = MultiLocation { parents: 1, interior: X1(Parachain(para_id)) };
-		T::Xcm::send_xcm(interior, dest, message).map_err(|e| match e {
-			SendError::CannotReachDestination(..) => Error::<T>::Unreachable,
-			_ => Error::<T>::SendFailure,
-		})?;
+		if !cfg!(feature = "runtime-benchmarks") {
```

Comment:
Can you create an issue about this please?

Reply:
yes, sure